src/core/qgsmessagelog.h

```diff
@@ -140,6 +140,9 @@ class CORE_EXPORT QgsMessageLogConsole : public QObject
   public:
     QgsMessageLogConsole();
 
+  protected:
+    QString formatLogMessage( const QString &message, const QString &tag, Qgis::MessageLevel level );
```

Comment:
I think that `formatLogMessage` should be a const method.

Reply:
Yep, will fix it.